translations/components.tex/es.po

```diff
@@ -259,33 +159,43 @@ msgid ""
 "    \\caption{\\textbf{\\pagelink{Town Actions}{\\mbox{Spell Book} Token}} \\iftoggle{printable}{}{\\phantom{Population}}}\n"
 "  \\end{subfigure}"
 msgstr ""
-"\\begin{minipage}[t]{0.48\\textwidth}\n"
+"[H]\n"
 "  \\centering\n"
-"  \\minipage[b]{0.5\\textwidth}\n"
+"  \\begin{subfigure}[t]{0.15\\linewidth}\n"
 "    \\centering\n"
-"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_fire.png}\n"
-"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Escuela del Fuego}}}\n"
-"  \\endminipage\n"
-"  \\minipage[b]{0.5\\textwidth}\n"
+"    \\includegraphics[width=0.8\\linewidth]{\\images/grail.png}\n"
+"    \\caption{\\textbf{\\pagelink{Grail}{Ficha del Grial} \\phantom{Population}}}\n"
+"  \\end{subfigure}\n"
+"  \\begin{subfigure}[t]{0.18\\linewidth}\n"
 "    \\centering\n"
-"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_water.png}\n"
-"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Escuela del Agua}}}\n"
-"  \\endminipage\n"
-"  \\hfill\\allowbreak%\n"
-"  \\bigbreak\n"
-"  \\minipage[b]{0.5\\textwidth}\n"
+"    \\includegraphics[width=\\linewidth]{\\images/gold-tokens.png}\n"
+"    \\caption{\\textbf{\\pagelink{Resources}{Ficha de Oro}}}\n"
+"  \\end{subfigure}\n"
+"  \\begin{subfigure}[c]{0.12\\linewidth}\n"
 "    \\centering\n"
-"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_air.png}\n"
-"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Escuela del Aire}}}\n"
-"  \\endminipage\n"
-"  \\minipage[b]{0.5\\textwidth}\n"
+"    \\includegraphics[width=0.6\\linewidth]{\\images/building-materials-token.png}\n"
+"    \\caption{\\textbf{\\pagelink{Resources}{Ficha de Material}}}\n"
+"  \\end{subfigure}\n"
+"  \\begin{subfigure}[c]{0.12\\linewidth}\n"
+"    \\centering\n"
+"    \\includegraphics[width=0.6\\linewidth]{\\images/valuables-token.png}\n"
+"    \\caption{\\textbf{\\pagelink{Resources}{Objetos de valor}}}\n"
+"  \\end{subfigure}\n"
+"  \\begin{subfigure}[c]{0.12\\linewidth}\n"
 "    \\centering\n"
-"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_earth.png}\n"
-"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Escuela de la Tierra}}}\n"
-"  \\endminipage\n"
-"  \\bigbreak\n"
-"\\end{minipage}\n"
-"\n"
+"    \\includegraphics[width=5em]{\\images/build.png}\n"
+"    \\caption{\\textbf{\\pagelink{Town Actions}{Construir}} \\iftoggle{printable}{}{\\phantom{Population}}}\n"
```

Comment:
Don't you want "Ficha de" here?

Reply:
If I put "Ficha de" there, the page will disarm, I need to put short names